c8run/README.md

```diff
@@ -4,10 +4,33 @@ C8 Run is a packaged distribution of Camunda 8, which allows you to spin up Camu
 
 Please refer to the [local installation with Camunda 8 Run guide](https://docs.camunda.io/docs/next/self-managed/setup/deploy/local/c8run/) for further details.
 
-## CI requirement for merging
+## Build C8Run locally
+
+### Prerequisites
+
+To successfully build `c8run`, you need:
+
+- Go 1.21 or later
+- Bash shell
+- OKTA credentials for internal Camunda artifacts
+
+### Build steps
 
-Only CI checks related to C8Run (those with "c8run" in the name) and CI runs marked as `required` are needed to merge. Non-C8Run-related CI checks can be ignored.
+Export your OKTA credentials as:
 
-## Build C8run locally
+```bash
+export JAVA_ARTIFACTS_USER=username
```

Comment:
It was not 100% clear to me which format this needs to be (tried a couple of times with the full email adress before only trying the local part):

```suggestion
export JAVA_ARTIFACTS_USER=firstname.lastname
```